src/core/src/resources/poet_config.toml

```diff
@@ -69,3 +69,8 @@ genesis_batch_path = ''
 # engine.
 
 validator_pub_key = '/etc/sawtooth/keys/validator.pub'
+
+# log_dir specifies the directory where the log file must be located
+# Note that the log filename would be poet-consensus.log( not configurable )
+ 
+log_dir = '/var/log/sawtooth'
```

Comment:
Sorry for late comment, didn't realize earlier.
"packaging/postint" file has given access to "/var/log/sawtooth-poet" folder for sawtooth user. Either change this to "/var/log/sawtooth-poet" or add "/var/log/sawtooth" folder to postint file.

Reply:
I was expecting this to be /var/log/sawtooth. Preferably sawtooth-poet would be a better choice as with roll-up a common directory for a setup running multiple components would be clumsy. Better have a segregation.